src/tests/routes/crates/read.rs

```diff
@@ -143,7 +137,7 @@ fn version_size() {
         .iter()
         .find(|v| v.num == "1.0.0")
         .expect("Could not find v1.0.0");
-    assert_eq!(version1.crate_size, Some(134));
+    assert_eq!(version1.crate_size, Some(158));
```

Comment:
similar to the comment above, these numbers changed because of the `description` and `license` fields that are now added to the `Cargo.toml` file by default